desktop-app/app/actions/browser.js

```diff
@@ -698,6 +716,23 @@ export function onDeviceMutedChange(deviceId, isMuted) {
   };
 }
 
+export function onAllDevicesDesignModeChange() {
+  return (dispatch: Dispatch, getState: RootStateType) => {
+    const {
+      browser: {allDevicesInDesignMode},
+    } = getState();
+    const next = !allDevicesInDesignMode;
+    pubsub.publish(TOGGLE_DEVICE_DESIGN_MODE_STATE, [{designMode: next}]);
+    dispatch(toggleAllDevicesDesignMode(next));
```

Comment:
Since it's a toggle, we don't need to pass a value as we retrieve the previous value from the state in the reducer.

Reply:
I think it's ok to change it